radix-engine/src/blueprints/native_schema.rs

```diff
@@ -22,9 +23,11 @@ lazy_static! {
         AccountNativePackage::definition();
     pub static ref IDENTITY_PACKAGE_DEFINITION: PackageDefinition =
         IdentityNativePackage::definition();
-    pub static ref ACCESS_CONTROLLER_PACKAGE_DEFINITION: PackageDefinition =
-        AccessControllerNativePackage::definition();
-    pub static ref POOL_PACKAGE_DEFINITION_V1_0: PackageDefinition =
+    pub static ref ACCESS_CONTROLLER_PACKAGE_DEFINITION_V1: PackageDefinition =
+        AccessControllerV1NativePackage::definition();
+    pub static ref ACCESS_CONTROLLER_PACKAGE_DEFINITION_V2: PackageDefinition =
+        AccessControllerV2NativePackage::definition();
+    pub static ref POOL_PACKAGE_DEFINITION_V2: PackageDefinition =
         PoolNativePackage::definition(PoolV1MinorVersion::Zero);
     pub static ref POOL_PACKAGE_DEFINITION_V1_1: PackageDefinition =
```

Comment:
I think we'd like consistent names for versions. Will follow up on Slack.

Reply:
Yup my bad, I think that this is a mistake, will correct.